BUILDING.md

```diff
@@ -19,7 +19,7 @@ sudo gem update
 
 3) Some dependencies are added via carthage. Run:
 ```
-carthage update
+carthage checkout
```

Comment:
Since we're shlepping our dependencies in a submodule, maybe the proper build instructions should be to run `git submodule update --init`